src/main/java/io/supertokens/cronjobs/deleteExpiredTotpTokens/DeleteExpiredTotpTokens.java

```diff
@@ -0,0 +1,50 @@
+package io.supertokens.cronjobs.deleteExpiredTotpTokens;
+
+import io.supertokens.Main;
+import io.supertokens.pluginInterface.STORAGE_TYPE;
+import io.supertokens.pluginInterface.totp.sqlStorage.TOTPSQLStorage;
+import io.supertokens.cronjobs.CronTask;
+import io.supertokens.cronjobs.CronTaskTest;
+import io.supertokens.storageLayer.StorageLayer;
+
+public class DeleteExpiredTotpTokens extends CronTask {
```

Comment:
You need to modify Main.java to add this cronjob.

Reply:
Done.